tools/e2e/README.md

```diff
@@ -0,0 +1,51 @@
+# Mainflux Users Groups Things and Channels E2E Testing Tool
+
+A simple utility to create a list of groups and users connected to these groups and channels and things connected to these channels.
+
+## Installation
+
+```bash
+cd tools/e2e
+make
+```
+
+### Usage
+
+```bash
+./e2e --help
+Tool for testing end to end flow of mainflux by creating groups and users and assigning the together andcreating channels and things and connecting them together.
+Complete documentation is available at https://docs.mainflux.io
+
+Usage:
+  e2e [flags]
+
+Flags:
+  -h, --help                   help for e2e
+  -H, --host string            address for mainflux instance (default "localhost")
```

Comment:
Explain what's `mainflux instance`.

Reply:
Add some more explantation (in the doc, no need to add it in help) about what `MF instance` means.